cke-tools/src/cmd/updateblock117/cmd/operate.go

```diff
@@ -0,0 +1,62 @@
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/cybozu-go/cke-tools/pkg/updateblock117"
+	"github.com/spf13/cobra"
+)
+
+var operateCmd = &cobra.Command{
+	Use:   "operate <block-pv-name>",
```

Comment:
I don't think using `<>` is appropriate for this.
```suggestion
	Use:   "operate PV",
```